.gitignore

```diff
@@ -13,3 +13,7 @@ test/*.cov
 site/
 /go-diagrams/
 *.idea/
+
+# client code for local testing only
```

Comment:
do we need to commit this to .gitignore?

Reply:
I have this file on my local for testing purpose... 
and I want to keep it.......

Reply:
you can override using your global gitignore